platform/security/handler/security-handler-oidc/src/main/java/org/codice/ddf/security/handler/oidc/OidcHandlerConfigurationImpl.java

```diff
@@ -128,7 +128,7 @@ public OidcClient getOidcClient(String callBackUri) {
     } catch (TechnicalException e) {
       LOGGER.warn(
```

Comment:
Might it be worthwhile to still log the error at a debug level in addition to the information at the warn level?

Reply:
I could see it being useful to have the full stack trace available somewhere